redBorder/n2kafka
# HTTP & Kafka tests

Use `curl` on testing to increment coverage.
- [x] Test 0014: Test `HTTP` return codes with `curl`
  - Add curl tests to check HTTP return values
- [ ] Test 0015: Test `HTTP` and `TCP` listeners  with `kafka` consumer.
  - Add tests for HTTP listener with kafka

---

Closes #7
